lib/presentation/contact_manager/contact_manager.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:dartz/dartz.dart';
+import 'package:fluffychat/app_state/failure.dart';
+import 'package:fluffychat/app_state/success_converter.dart';
+import 'package:fluffychat/app_state/success.dart';
+import 'package:fluffychat/config/app_config.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/domain/app_state/contact/get_contacts_state.dart';
+import 'package:fluffychat/domain/usecase/get_all_contacts_interactor.dart';
+import 'package:fluffychat/domain/usecase/search_contacts_interactor.dart';
+import 'package:fluffychat/presentation/converters/presentation_contact_converter.dart';
+import 'package:fluffychat/presentation/extensions/contact/presentation_contact_extension.dart';
+import 'package:fluffychat/presentation/model/presentation_contact.dart';
+import 'package:flutter/material.dart';
+import 'package:matrix/matrix.dart';
+
+class ContactManager {
+  bool _firstSynchronize = false;
+
+  final _getAllContactsInteractor = getIt.get<GetAllContactsInteractor>();
+
+  final searchContactsInteractor = getIt.get<SearchContactsInteractor>();
+
+  final SuccessConverter converter = PresentationContactConverter();
+
+  // FIXME: Consider can use FocusNode instead ?
+  final ValueNotifier<bool> isSearchModeNotifier = ValueNotifier(false);
+
+  final ValueNotifier<Either<Failure, Success>> contactsNotifier =
+      ValueNotifier<Either<Failure, Success>>(
+    const Right(GetContactsInitial()),
+  );
+
+  final ValueNotifier<List<PresentationContact>> tomPresentationContacts =
+      ValueNotifier<List<PresentationContact>>([]);
+
+  final ValueNotifier<List<PresentationContact>> phonebookPresentationContacts =
+      ValueNotifier<List<PresentationContact>>([]);
+
+  void initialSynchronizeContacts() {
+    if (!_firstSynchronize) {
+      getAllContacts();
+      _firstSynchronize = !_firstSynchronize;
+    }
+  }
+
+  void getAllContacts({String? keyword}) {
```

Comment:
```suggestion
  void _getAllContacts() {
```
it should be private function without keyword
search contact should be handle by screen controller

Reply:
Sure 
- Remove keyword property  not use
- private func